homeassistant/components/unifi/strings.json

```diff
@@ -33,6 +33,27 @@
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
   },
+  "entity": {
+    "sensor": {
+      "device_state": {
+        "state": {
+          "disconnected": "Disconnected",
+          "connected": "Connected",
```

Comment:
Couldn't some states have references to already existing translations?

Reply:
Good point, forgot about that.